server/lobby.js

```diff
@@ -392,6 +392,29 @@ class Lobby {
             return;
         }
 
+        if(gameDetails.quickJoin) {
+            let sortedGames = _.sortBy(this.games, 'createdAt');
+            let gameToJoin = sortedGames.find(game => !game.started && game.gameType === gameDetails.gameType && _.size(game.players) < 2 && !game.password);
```

Comment:
This is probably fine for now but it might be nice to encapsulate the logic on the game object (particularly for Melee purposes where that size check wouldn't be correct)

Reply:
Yes, this needs to be game.canJoin(), but that refactor is too big for this small change right now